Resources/doc/index.rst

```diff
@@ -327,6 +344,33 @@ Let's walk through this example:
   to MongoDB. In this example, the ``$product`` object has not been persisted yet,
   so the document manager makes a query to MongoDB, which adds a new entry.
 
+If you are using autowiring, you can use type hinting to fetch the doctrine_mongodb.odm.document_manager service::
```

Comment:
Should `doctrine_mongodb.odm.document_manager` get monospace treatment with double backticks?

Reply:
Since `autowiring` is already defined as a link, would it make sense to link this instance of the word as well (as you did on line 49 in this diff)?